README.md

```diff
@@ -249,8 +254,8 @@ function ConfigRegistry(config){
 util.inherits(ConfigRegistry, DefaultRegistry);
 
 ConfigRegistry.prototype.set = function set(name, fn) {
-  var task = this._tasks[name] = fn.bind(this.config);
-  return task;
+  fn = fn.bind(this.config);
+  return DefaultRegistry.prototype.set.call(this, name, fn) || fn;
```

Comment:
Like above, my aim here was future proofing. And thinking users shouldn't have to care about default registry specifics. Agreed it isn't very pretty though! 

Really though I wouldn't expect the kind of users who might implement custom registries to be frightened by this. So short of syntactic sugar coating for super calls, I'd say it's either this or expose default registry guts like _tasks.

But fair enough, I'll amend the PR!